### ISSUE TITLE
Support for configuring IPv6 networks

### ISSUE THREAD

**Issue by jawnsy** [terraform-google-modules/terraform-google-network#338]
### TL;DR

This module currently configures IPv4 networks, and cannot be used to manage dual-stack networks

### Terraform Resources

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_subnetwork

### Additional information

It looks like we just need to add a few fields:

* private_ipv6_google_access
* stack_type
* ipv6_access_type

We would also want to make sure that we export the following outputs:

* ipv6_cidr_range
* external_ipv6_prefix

The changes seem contained to the `subnets` module: https://github.com/terraform-google-modules/terraform-google-network/blob/a7b91bc1a70c323b99eecc229a70b449b6bda401/modules/subnets/main.tf#L28-L33

**Comment by jawnsy** [comment]
This is still an issue, not stale :)

**Comment by bharathkkb** [comment]
Happy to review a PR adding this

**Comment by jlkiri** [comment]
it looks like `ipv6_cidr_range` attribute is already in the documentation but it is always empty for me when I try to use it

**Comment by Lynty** [comment]
@bharathkkb would you be able to review https://github.com/terraform-google-modules/terraform-google-network/pull/420? thank you!

**Comment by imrannayer** [comment]
fixed in #420

**Comment by jawnsy** [comment]
Thanks for doing this @Lynty and thanks for reviewing/merging @apeabody!
